pkg/config/exec_config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/warptools/warpforge/pkg/formulaexec"
+	"github.com/warptools/warpforge/pkg/plotexec"
+	"github.com/warptools/warpforge/pkg/workspace"
+)
+
+func BinPath(state State) string {
```

Comment:
Not really clear to me why this takes State as a parameter rather than being a method

Reply:
Dunno. Personal whims at the time.